public/locales/en/foundation.json

```
@@ -1,8 +1,15 @@
 {
   "title": "The Nervos Foundation.",
-  "slogan": "The Nervos Foundation is a non-profit organization whose mission is to support the long-term growth of the Nervos Network. We work to ensure the future success of Nervos by seeding the community to grow independently while driving meaningful innovation in the industry.",
+  "foundation": "Foundation",
+  "slogan": "<tag1>The Nervos Foundation</tag1> is a non-profit organization whose mission is to support the long-term growth of the <tag2>Nervos Network</tag2>. We work to ensure the future success of network by seeding the community to grow independently while driving meaningful innovation in the industry.",
```

```suggestion
  "slogan": "<tag1>The Nervos Foundation</tag1> is a non-profit organization whose mission is to support the long-term growth of the <tag2>Nervos Network</tag2>. We work to ensure the future success of the network by seeding the community to grow independently while driving meaningful innovation in the industry.",
```

> change "success of Nervos" to "success of the network"

Ref: https://github.com/Magickbase/nervos-official-website/issues/439#issuecomment-2006298757